omod/src/main/resources/config.xml

```diff
@@ -97,6 +98,15 @@
 	            </module>
 	        </modules>
 	    </conditionalResource>
+		<conditionalResource>
+			<path>/lib/patientflags-fhir.*</path>
+			<modules>
+				<module>
+					<moduleId>fhir2</moduleId>
+					<version>1.9.0</version>
```

Comment:
I think you could use `${fhirversion}` here instead and have that correctly filled in.

Reply:
I changed hard coded fhir version to `${fhirversion}`